en/documentation/index.md

```diff
@@ -18,48 +18,56 @@ The authoritative Ruby language documentation at [docs.ruby-lang.org][docs-rlo-e
 
 ### Getting Started
 
-[Official FAQ](/en/documentation/faq/)
-: The official frequently asked questions.
-
-[Ruby Koans][2]
-: The Koans walk you along the path to enlightenment in order to learn
-  Ruby. The goal is to learn the Ruby language, syntax, structure, and
-  some common functions and libraries. We also teach you culture.
-
-[Why’s (Poignant) Guide to Ruby][5]
-: An unconventional but interesting book that will teach you Ruby
-  through stories, wit, and comics. Originally created by *why the lucky
-  stiff*, this guide remains a classic for Ruby learners.
-
-[Ruby Essentials][7]
-: A free on-line book designed to provide a concise
-  and easy to follow guide to learning Ruby.
+[Try Ruby][1]
+: You can try Ruby right in your browser.
 
 [Learn to Program][8]
 : A wonderful little tutorial by Chris Pine for programming newbies. If
   you don’t know how to program, start here.
 
-[Learn Ruby the Hard Way][38]
-: A very good set of exercises with explanations that guide you from
-  the absolute basics of Ruby all the way to OOP and web development.
+[Ruby in Twenty Minutes][rubyin20]
+: A small Ruby tutorial that should take no more than 20 minutes to
+  complete.
 
-### Manuals
+[The Odin Project][odin]
+: An open source full stack curriculum
 
-[Programming Ruby][9]
-: The seminal work on Ruby in English, this first edition of the
-  [Pragmatic Programmers’ book][10] is available for free online.
+[excercism][exercism]
+: 120 exercises with automatic analysis and personal mentoring.
 
-[The Ruby Programming Wikibook][12]
-: A free online manual with beginner and intermediate content plus a
-  thorough language reference.
+[Codecademy][codecademy]
+: Online code bootcamp with a variety of topics. Freeish.
+
+### Manuals / Books
+
+#### Beginner
+
+[Programming Ruby 3.3][pickaxe]
+: The seminal work on Ruby in English. Recently updated to ruby 3.3.
+
+[The Well-Grounded Rubyist][grounded]
+: A tutorial that begins with your first Ruby program and takes you all the way to sophisticated topics like reflection, threading, and recursion.
+
+#### Adept
```

Comment:
As a non-native English speaker, I think `Adept` is a relatively rare choice of word. Maybe `Advanced` will be more approachable?

Reply:
Maybe `Experienced` would be better? Advanced and Expert are closer in my mind than Adept/Experienced and Expert are.

Reply:
Intermediate?

Reply:
Intermediate sounds good to me 👍